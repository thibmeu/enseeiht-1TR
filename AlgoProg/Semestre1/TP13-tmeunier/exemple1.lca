add x 1
value x
print
end
