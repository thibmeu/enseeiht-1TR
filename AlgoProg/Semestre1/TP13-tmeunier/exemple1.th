size 5
hash taille
add x 1
value x
print
add xx 2
print
add xxx 3
print
add xxxx 4
print
add xxxxx 5
print
end
