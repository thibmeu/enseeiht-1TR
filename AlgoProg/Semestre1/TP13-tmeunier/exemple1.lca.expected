> add x 1
> value x
1
> print
-->[ "x"=1 ]--E
> end
